en/includes/guides/customize/actions/setting-up-actions.md

```diff
@@ -70,3 +70,52 @@ The following are some of the troubleshooting steps that you may take to resolve
 1. Ensure that your external service is up and running and that there are no connectivity issues.
 
 2. Confirm that the request and response payloads conform to the expected formats as defined by the REST API contract.
+
+## Configuring HTTP Client Connections
```

Comment:
Shall we say 'Fine tune HTTP Client Connections'

Reply:
Updated in